app/services/bulk_update/insert_all.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+module BulkUpdate
+  class InsertAll
```

Comment:
Nitpick: Is a better name for this class `UpsertAll` because I don't think we ever insert with bulk update ?